packages/generator/src/generator-without-ts-morph/service/class.spec.ts

```diff
@@ -24,27 +24,20 @@ describe('class', () => {
     expect(result).not.toContain('actionImports');
   });
 
-  it('contains functionImports if in VDM', () => {
+  it('contains operations if in VDM', () => {
     const result = serviceClass({
       ...service,
       functionImports: [
         { name: 'myFunction', parametersTypeName: 'paraName' } as any
+      ],
+      actionImports: [
+        { name: 'myAction', parametersTypeName: 'paraName' } as any
```

Comment:
I think this is because `VdmServiceMetadata` separates `actionImports` and `functionImports` now. Should these be merged as well, possibly in a follow-up?

Reply:
I agree. I'll create a ticket for this.